main.py

```diff
@@ -38,42 +40,76 @@ def data_key_to_file_name(data_key):
     return file_name, extension
 
 
-def main():
-    data_elements = sds.SedohDataElements().data_elements
-
-    # elements = sds.SedohDataElements().data_elements
-    # Setup: Load data files for data sources that do not have an existing API
-    print(f"Importing data files")
-    data_files = load_data_files()
+def input_file_validation(data_frame, version):
+    if version in [None, 1, 2]:
```

Comment:
This if statement can be removed. Please put in a comment saying something along the lines that ALL versions require these columns to be present.